enterprise/server/cmd/ci_runner/main.go

```diff
@@ -71,6 +71,10 @@ const (
 	bazeliskBinaryName             = "bazelisk"
 	bazelCommandOverrideEnvVarName = "BAZEL_COMMAND"
 
+	// Git
+
+	remoteName = "origin"
```

Comment:
maybe call this remoteOrigin instead so the callsites where you use it are clearer?